src/kv/test/kv_serialisation.cpp

```diff
@@ -324,12 +325,18 @@ TEST_CASE("Integrity" * doctest::test_suite("serialisation"))
 
     // Here, a real encryptor is needed to protect the integrity of the
     // transactions
-    auto secrets = std::make_shared<ccf::LedgerSecrets>();
-    secrets->init();
-    auto encryptor = std::make_shared<ccf::RaftTxEncryptor>(1, secrets);
+    std::list<kv::TxEncryptor::KeyInfo> keys;
+    std::vector<uint8_t> raw_key(crypto::GCM_SIZE_KEY);
+    for (size_t i = 0; i < raw_key.size(); ++i)
+    {
+      raw_key[i] = i;
+    }
+    keys.push_back({kv::Version(0), raw_key});
+    auto encryptor = std::make_shared<kv::TxEncryptor>(keys);
```

Comment:
Side effect of the stripping of `Encryptor` dependencies. The tests don't need to create a 'real' `LedgerSecrets`, just a "here's the key to use from version 0" list.